
Currently Loaded Modules:
  1) anaconda/anaconda-2023.09   2) cuda/cuda-12.2.2

 

mkdir: cannot create directory ‘362631’: File exists
Traceback (most recent call last):
  File "./discrete/run/teacher/dungeon_quest_teacher_rew_per_step_7_productMDP.py", line 5, in <module>
    from discrete.lib.main import run_training
  File "/lustre/fs1/home/amckinney/neuroSym/Local_Neuro_Symbolic/discrete/lib/main.py", line 11, in <module>
    from discrete.lib.training import train_agent, distill_agent
  File "/lustre/fs1/home/amckinney/neuroSym/Local_Neuro_Symbolic/discrete/lib/training.py", line 879
    if if ep_counter+1 % 1000 == 0:
        ^
SyntaxError: invalid syntax
mv: cannot stat './test_output/test_output*': No such file or directory
Traceback (most recent call last):
  File "./discrete/run/target/dungeon_quest_7_target_rew_per_step_productMDP_DDPG.py", line 10, in <module>
    from discrete.lib.main import run_training
  File "/lustre/fs1/home/amckinney/neuroSym/Local_Neuro_Symbolic/discrete/lib/main.py", line 11, in <module>
    from discrete.lib.training import train_agent, distill_agent
  File "/lustre/fs1/home/amckinney/neuroSym/Local_Neuro_Symbolic/discrete/lib/training.py", line 879
    if if ep_counter+1 % 1000 == 0:
        ^
SyntaxError: invalid syntax
mv: cannot stat './test_output/test_output*': No such file or directory
mv: cannot stat 'err.job.362631.err': No such file or directory
mv: cannot stat 'out.job.362631.out': No such file or directory
